src/layouts/flex/lv_flex.h

```diff
@@ -22,9 +22,9 @@ extern "C" {
  *      DEFINES
  *********************/
 
-#define _LV_FLEX_COLUMN        (1 << 0)
-#define _LV_FLEX_WRAP       (1 << 2)
-#define _LV_FLEX_REVERSE    (1 << 3)
+#define LV_FLEX_COLUMN        (1 << 0)
```

Comment:
If remove the underscrore let's `#undef` these after `lv_flex_flow_t`.

Reply:
Hmm, I agree but the .c file depends on these defines. And I can't put them in a private header because this public header needs them to define the enum.